coderd/coderd.go

```diff
@@ -984,6 +984,8 @@ func New(options *Options) *API {
 				// This value is intentionally increased during tests.
 				r.Use(httpmw.RateLimit(options.LoginRateLimit, time.Minute))
 				r.Post("/login", api.postLogin)
+				r.Post("/request-one-time-passcode", api.postRequestOneTimePasscode)
```

Comment:
Not sure if somebody touched naming case, here are my suggestions:

`/otp/request`
`/otp/change-password`

Reply:
I'm happy with this. So to confirm the full path would be `/api/v2/users/otp/request` and `/api/v2/users/otp/change-password`?

Reply:
Yes! Same way we have `/api/v2/users/oidc` or `/api/v2/users/oauth2`